INSTALLATION.md

```diff
@@ -0,0 +1,59 @@
+# Installation Guide
+
+We welcome users to install and try our Hpypertrons Chrome Extension. If you have any feedback, please post it in the issue area (with description of the problem and browser configurations). We will actively respond and solve it!
```

Comment:
`Hpypertrons` should be `Hypertrons` here.

Reply:
Now we have provided the `hypertrons.zip`, so you can update the procedure `unzip hypertrons.crx`.

Reply:
All right! problems fixed.